lib/client.js

```diff
@@ -159,6 +159,9 @@ class Client extends EventEmitter {
       }
     }
 
+    // Adding a unique request id to the request
+    data.$req_id = crypto.randomUUID();
```

Comment:
Maybe not in this PR, but we should support the ability to pass in an external `req_id` with the request.  It could be useful for end-to-end tracing in apps / clients / storefronts.

Reply:
It could be helpful, but at the moment `client.request` interface already has `data` which becomes `$data`, so we need add additional logic for parsing headers, I think we can implement it a bit later